aspnet/web-forms/overview/deployment/configuring-server-environments-for-web-deployment/choosing-the-right-approach-to-web-deployment/samples/sample7.cmd

```diff
@@ -3,8 +3,7 @@ msdeploy.exe
   -dest:auto,
         computerName='https://STAGEWEB1:8172/MSDeploy.axd?site=DemoSite',
         userName='FABRIKAM\stagingdeployer',
-        password='Pa$$w0rd',
         authtype='Basic', 
```

Comment:
Need the NTLM flag

Reply:
Just a note on where we ended up on this one.  We decided to back to Basic auth plus the placeholder  $CREDENTIAL_PLACEHOLDER$